sections/content_fullTable/fullTable.R

```diff
@@ -68,15 +70,15 @@ output$fullTable <- renderDataTable({
     selection = "none",
     options   = list(
       pageLength     = -1,
-      order          = list(11, "desc"),
+      order          = list(12, "desc"),
       scrollX        = TRUE,
       scrollY        = "calc(100vh - 250px)",
```

Comment:
Maybe we can adjust the height of the table a bit so that the slider is visible all the time?
```suggestion
      scrollY        = "57vh",
```

Reply:
ok